packages/flutter/lib/src/cupertino/button.dart

```diff
@@ -329,6 +342,11 @@ class _CupertinoButtonState extends State<CupertinoButton> with SingleTickerProv
       _buttonHeldDown = false;
       _animate();
     }
+    final RenderBox rb = context.findRenderObject()! as RenderBox;
```

Comment:
Type the name out per the styleguide: https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#avoid-abbreviations